### PR TITLE
K8SPSMDB-403 fix: some typos in comments/messages

### PR DESCRIPTION
**Pull request by ollevche**
[K8SPSMDB-403](https://jira.percona.com/browse/K8SPSMDB-403) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://jira.percona.com/browse/K8SPSMDB-403

### CODE REVIEW

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| demand-backup-sharded | passed |
| version-service | passed |
| users | passed |
| upgrade | failed |
| rs-shard-migration | passed |
| data-sharded | passed |

File: pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func (r *ReconcilePerconaServerMongoDBBackup) Reconcile(request reconcile.Reques
 	return rr, nil
 }
 
-// reconcile backup. first we chek if there are concurrent job running
+// reconcile backup. first we check if there are concurrent job running
```
Comment (hors) [inline pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go:133]:
```suggestion
// reconcile backup. firstly we check if there are concurrent jobs running
```

File: pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go [hunk 2]
```diff
@@ -251,7 +251,7 @@ func (r *ReconcilePerconaServerMongoDBRestore) getBackup(cr *psmdbv1.PerconaServ
 func (r *ReconcilePerconaServerMongoDBRestore) updateStatus(cr *psmdbv1.PerconaServerMongoDBRestore) error {
 	err := r.client.Status().Update(context.TODO(), cr)
 	if err != nil {
-		// may be it's k8s v1.10 and erlier (e.g. oc3.9) that doesn't support status updates
+		// may be it's k8s v1.10 and earlier (e.g. oc3.9) that doesn't support status updates
```
Comment (hors) [inline pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go:254]:
```suggestion
		// maybe it's k8s v1.10 and earlier (e.g. oc3.9) that doesn't support status updates
```

**Review by hors — CHANGES_REQUESTED** [review]
please see comments

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-570-d85aa188

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| scheduled-backup | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| demand-backup-sharded | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| users | passed |
| data-sharded | passed |

**Review by delgod — APPROVED** [review]

**Pull request merged**